# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/l10n/tests/mochitest/dom_localization/test_getAttributes.html
# Commit: 595529cd906f
# Full Hash: 595529cd906f94cdb2bc12a7a40072db0c07198e
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-03 22:01:19
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part9: Update DOMLocalization to match Localization constructor. r=nika,emilio
#   
#   Depends on D113691
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D113692
# ==============================================================================

diff -r 9f3e2963d925 -r 595529cd906f dom/l10n/tests/mochitest/dom_localization/test_getAttributes.html
--- a/dom/l10n/tests/mochitest/dom_localization/test_getAttributes.html	Tue Aug 03 05:52:03 2021 +0000
+++ b/dom/l10n/tests/mochitest/dom_localization/test_getAttributes.html	Tue Aug 03 05:52:03 2021 +0000
@@ -8,15 +8,12 @@
   <script type="application/javascript">
   "use strict";
 
-  async function* generateBundles(resourceIds) {}
-
   window.onload = function() {
     SimpleTest.waitForExplicitFinish();
 
     const domLoc = new DOMLocalization(
       [],
       false,
-      { generateBundles },
     );
 
     const p1 = document.querySelectorAll("p")[0];